src/layouts/Root/components/Footer.tsx

```diff
@@ -16,6 +16,10 @@ import MaxWidthLayout from 'layouts/MaxWidth';
 const StyledFooterItem = styled(Link)`
   margin-right: ${p => p.theme.space.lg};
   color: ${p => p.theme.palette.white};
+
+  &:hover {
```

Comment:
Should you account for `&:focus` as well?

Reply:
Ah yes, I forgot about focus, thanks!